### ISSUE TITLE
Association stats box is cut off 

### ISSUE THREAD

**Issue by prashantiA** [googleinterns/step126-2020#8]
Text is cut off on the association stats box

**Comment by nitinramadoss** [comment]
Using margin: 0 auto may help with centering and keeping the text within the container

**Comment by prashantiA** [comment]
Yeah I think a big part of the issue is the text size doesn't dynamically resize so i don't want to just make the font smaller because then then it will look weird on big monitors. I'll look into how to fix this at a later date (leaving it here for now so I don't forgot).
